addons/survey/static/tests/tours/certification_failure.js

```diff
@@ -1,6 +1,17 @@
 odoo.define('survey.tour_test_certification_failure', function (require) {
 'use strict';
 
+var SurveyFormWidget = require('survey.form');
+/**
+ * Speed up fade-in fade-out to avoid useless delay in tests.
```

Comment:
"useless delay" ... we all know what you mean !

Reply:
😇